style.css

```diff
@@ -0,0 +1,268 @@
+* {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+body {
+    margin: 0 auto;
+    height:100%;
```

Comment:
please check your formatting. you need to keep it consistent. spaces after : before values

Reply:
made consistent in commit 1cd39ee